PlayFabClientSDK/source/PlayFabHttp/PlayFabHttp.cs

```diff
@@ -41,28 +41,15 @@ public class PlayFabJsonSuccess<TResult> where TResult : PlayFabResultCommon
 
     public static class PlayFabHttp
     {
-        private static ITransportPlugin _transport;
-
         public static async Task<object> DoPost(string urlPath, PlayFabRequestCommon request, string authType, string authKey, Dictionary<string, string> extraHeaders)
```

Comment:
Keep AuthType and AuthKey here for now. We have to have them somewhere, so this is the top.
But nothing else, anywhere else should have them.